src/test/java/eu/dissco/core/digitalspecimenprocessor/utils/TestUtils.java

```diff
@@ -38,6 +38,7 @@ public class TestUtils {
   public static String SOURCE_SYSTEM_ID = "20.5000.1025/MN0-5XP-FFD";
   public static JsonNode ORIGINAL_DATA = generateSpecimenOriginalData();
   public static String DWCA_ID = null;
+  public static String TOPIC_DISCIPLINE = "geology";
```

Comment:
Is this used? The topic discipline for geology is "earth systems" (geology is not a topic discipline)